code/modules/antagonists/abductor/equipment/abduction_gear.dm

```diff
@@ -216,18 +216,21 @@
 	. = ..()
 	if(flag)
 		return
+	. |= AFTERATTACK_PROCESSED_ITEM
```

Comment:
seems inconsistent with the flamethrower check, what's the pattern?

Reply:
flamethrower clicking on an item is always reasonable because you might just be trying to fire in its direction

for this i have no idea what the fuck it is so i can fix it to be like flamethrower

Reply:
actually this one is completely irrelevant because the only stuff that could ever care about this also requires proximity flag. after attack is shit and we need a ranged attack. just going to remove this and everything like it and document that only proximity flag matters